ts/api/api_test.ts

```diff
@@ -70,6 +71,19 @@ describe('OpenYolo API', () => {
         });
       });
 
+      describe('cancelLastRequest', () => {
```

Comment:
There's also the equivalent test without timeout to add further down in the file, for completeness.